docs/src/development/local/ddev.md

```diff
@@ -104,7 +99,7 @@ ddev pull platform
 To skip pulling files, add `--skip-files` to the command.
 To skip pulling a database, add `--skip-db` to the command.
 
-## 6. (Optional) Install dependencies
+## 6. Optional: Install dependencies
```

Comment:
Using the add-on, this part should not be necessary as build/deploy hooks are supported.

Reply:
Agreed, I don't think there's any need for installing extra things with composer in this context.